my-app/src/joinGamePage.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+function JoinGamePage(){
+    return (
+        
+        <div>
```

Comment:
```suggestion
import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';

function JoinGamePage(){
    return (
        <div>
```

minor spacing and formating